out/build/x64-Debug/VSInheritEnvironments.txt

```diff
@@ -0,0 +1 @@
+msvc_x64_x64
```

Comment:
There's still accidentally commited files left in.

Reply:
Removed